package.json

```diff
@@ -11,10 +11,13 @@
   },
   "dependencies": {
     "@mobileaction/action-kit": "^1.29.0",
+    "autoprefixer": "^10.4.19",
     "bootstrap-select": "^1.13.18",
     "mdb-ui-kit": "^7.3.2",
+    "postcss": "^8.4.39",
     "primeicons": "^7.0.0",
     "primevue": "^4.0.0",
+    "tailwindcss": "^3.4.4",
     "vue": "^3.4.29",
     "vue-router": "^4.3.3"
```

Comment:
Configure browserlists to support only last 2 years browsers, See [this](https://github.com/browserslist/browserslist)
You can simply add this like
` "browserslist": [
    "last 2 years"
  ],`